helm/openwhisk/values.yaml

```diff
@@ -248,8 +254,8 @@ nginx:
 
 # Controller configurations
 controller:
-  imageName: "openwhisk/controller"
-  imageTag: "c5970a6"
+  imageName: "hunhoffe/controller"
```

Comment:
TODO: need to update this once apache/openwhisk has accepted changes needed for controller to replicate properly